dist/index.html

```diff
@@ -5,88 +5,101 @@
     <title>Seek-A-Design Playground</title>
     <link rel="stylesheet" href="style.css">
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
-    <script src="../src/index.js"></script>
     <script src="https://kit.fontawesome.com/9a26908d19.js"></script>
+    <script src="../src/index.js"></script>
   </head>
   <body>
     <div class="container-parent">
       <div class="container-header">
         <div class="container-logo">
-            <img src="https://i.ibb.co/47tS3wH/seeklogo.png" alt="seeklogo" border="0" id="mainlogo">
+          <img src="https://i.ibb.co/47tS3wH/seeklogo.png" alt="seeklogo" id="mainlogo">
         </div>
         <div class="container-navSection">
           <div class="container-nav">
-                <div class="menu"> <a href="home.asp" class="fas fa-home">Home</a> </div>
-                <div class="menu"> <a href="projects.asp" class="fas fa-desktop">Projects</a> </div>
-                <div class="menu"> <a href="about.asp" class="fas fa-user">About_Me</a> </div>
-                <div class="menu"> <a href="contact.asp" class="fas fa-envelope">Contact</a> </div>
+
+            <div class="menu">
+              <span class="fas fa-home"></span><p>Home</p>
+            </div>
+            <div class="menu">
+              <span class="fas fa-desktop"></span><p>Projects</p>
+            </div>
+            <div class="menu">
+              <span class="fas fa-user"></span><p>About Me</p>
+            </div>
+            <div class="menu">
+                <span class="fas fa-envelope"></span><p>Contact</p>
+            </div>
+
           </div>
           <div class="container-moto">
-            <h2>Custom Designs · Marketing · Branding · And More </h2>
+            <h2>Custom Designs · Marketing · Branding · And More</h2>
           </div>
         </div>
       </div>
       <div class="container-body">
         <div class="content">
           <div class="content-left">
-            <span class ="ownerContact" style="color: #336D61; font-size: 3em; font-family: AlexandriaFLF;">
+            <span class="ownerContact">
               CONTACT
             </span>
-            </h2>
-              <hr>
-            <p style="text-align: center;">
-              <span class="ownerName">
-                LYANDRA SICA<br/>
-              </span>
+            <!-- TODO: Replace with static horizontal rule asset when delivered by designer. -->
+            <hr>
+              <div class="ownerTitle">
+                <span class="ownerName">
+                  LYANDRA SICA<br/>
```

Comment:
I still think we could use CSS to get rid of these `<br/>` tags by taking advantage of CSS properties like `margin` and `padding`.

Reply:
I won't litter your PR with similar comments. Just note that I believe this for all instances of `<br/>` tags in this file.

Reply:
removed all `<br/>` tags